src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/controller/dto/CreateRecallResponse.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.controller.dto
+
+import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.jpa.entity.RecallEntity
+import java.util.UUID
+
+data class CreateRecallResponse(
+  val recallUuid: UUID,
+) {
+  companion object {
+    fun transform(recall: RecallEntity): CreateRecallResponse =
```

Comment:
minor point, the other dtos have a `from` method rather than `transform`
```suggestion
    fun from(recall: RecallEntity): CreateRecallResponse =
```